collections/README.md

```diff
@@ -24,18 +24,23 @@ aim to align with it. But it still seems to be in flux.*
-| `data`         | `/collections` | OAFeat         | List of Collections                                                                                                                                                                                                                                            |
+| **rel**        | **href**       | **From**  | **Description**                  |
+| -------------- | -------------- | --------- | -------------------------------- |
+| `root`         | `/`            | STAC Core | The root URI                     |
+| `self`         | `/`            | OAFeat    | Self reference, same as root URI |
+| `service-desc` | `/api`         | OAFeat    | The service description          |
+| `data`         | `/collections` | OAFeat    | List of Collections              |
+
+The path for the `service-desc` endpoint is recommended to be `/api`, but may be another path. Recommended to be
+OpenAPI 3.0 or 3.1 with media types `application/vnd.oai.openapi` (YAML),
+`application/vnd.oai.openapi+json;version=3.0` (3.0 JSON), or `application/vnd.oai.openapi+json;version=3.1`
+(3.1 JSON).
 
 A `service-doc` endpoint is recommended, but not required.
 
-| **rel**       | **href**    | **From**       | **Description**                                                                                                                                                                                                     |
-| ------------- | ----------- | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `service-doc` | `/api.html` | OAFeat OpenAPI | An HTML service description.  Uses the `text/html` media type to refer to a human-consumable description of the service. The path for this endpoint is only recommended to be `/api.html`, but may be another path. |
+| **rel**       | **href**    | **From** | **Description**                                                                                                                                                                                                     |
+| ------------- | ----------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `service-doc` | `/api.html` | OAFeat   | An HTML service description.  Uses the `text/html` media type to refer to a human-consumable description of the service. The path for this endpoint is only recommended to be `/api.html`, but may be another path. |
```

Comment:
What does "uses text/html" mean? Is that required? Or is that one of the allowed values? Wondering whether I could also use xslt, xhtml or other media types that a browser can render flawlessly instead of HTML?

Reply:
We've always had in this doc that it had to be text/html, but now that I look at the OAFeat Part 1 spec more closely, it doesn't require that it be such. I'll update this to reflect that.

Reply:
updated.